draft-ietf-httpbis-binary-message.md

```diff
@@ -147,7 +147,9 @@ a header section with a length prefix, binary content with a length prefix, and
 a trailer section with a length prefix.
 
 Response messages that contain informational status codes result in a different
-structure; see {{informational}}.
+structure; see {{informational}}.  Note that while the Known-Length
+Informational Response field is shown in {{format-known-length}}, it is only
+permitted in response messages.
```

Comment:
"permitted" isn't the right word - this is about a field that just cannot be present, not about an optional field that is not allowed to be present?

Reply:
s/it is only permitted in/it cannot appear in/ perhaps?

Reply:
Or "it only appears". I still think a separate diagram would make this easier to read personally though